hardware_interface/include/hardware_interface/internal/interface_manager.h

```diff
@@ -142,12 +162,23 @@ class InterfaceManager
   /**
    * \brief Get an interface.
    *
-   * Since this class only stores one interface per type, this returns a
-   * pointer to the requested interface type. If the interface type is not
-   * registered, it will return \c NULL.
+   * If this class and its registered sub-managers only have one registered
+   * instance of the requested type, this will be returned. If multiple
+   * instances of the interface type were registered and the type is a
+   * \c ResourceManager, this call will try to combine them into a single
+   * handle. In all other cases, \c nullptr will be returned.
+   *
+   * \note As this instance will only store one instance of each manager type,
+   * the only way multiple interfaces are registered is the registration of
+   * other managers with interfaces of the same type.
+   *
+   * \note If there are multiple registered interfaces of the requested type
+   * (either directly or through registered interfaces) and they are not
```

Comment:
Not sure this "(either directly or through registered interfaces)" is needed. If anything, it should say "registered sub-managers", but I think the rest of this block comment describes the need for sub-managers already.

Reply:
Sure, applied in 04bcb42a80da3b7f6e8dc498532723ab8ba00fee